templates/frontend/objects/chapter.tpl

```diff
@@ -0,0 +1,501 @@
+{**
+ * templates/frontend/objects/chapter.tpl
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Display a full view of a chapter. Expected to be primary object on
+ *  the page.
+ *
+ * Many presses will need to add custom data to this object, either through
+ * plugins which attach to hooks on the page or by editing the template
+ * themselves. In order to facilitate this, a flexible layout markup pattern has
+ * been implemented. If followed, plugins and other content can provide markup
+ * in a way that will render consistently with other items on the page. This
+ * pattern is used in the .main_entry column and the .entry_details column. It
+ * consists of the following:
+ *
+ * <!-- Wrapper class which provides proper spacing between components -->
+ * <div class="item">
+ *     <!-- Title/value combination -->
+ *     <div class="label">Abstract</div>
+ *     <div class="value">Value</div>
+ * </div>
+ *
+ * All styling should be applied by class name, so that titles may use heading
+ * elements (eg, <h3>) or any element required.
+ *
+ * <!-- Example: component with multiple title/value combinations -->
+ * <div class="item">
+ *     <div class="sub_item">
+ *         <div class="label">DOI</div>
+ *         <div class="value">12345678</div>
+ *     </div>
+ *     <div class="sub_item">
+ *         <div class="label">Published Date</div>
+ *         <div class="value">2015-01-01</div>
+ *     </div>
+ * </div>
+ *
+ * <!-- Example: component with no title -->
+ * <div class="item">
+ *     <div class="value">Whatever you'd like</div>
+ * </div>
+ *
+ * Core components are produced manually below, but can also be added via
+ * plugins using the hooks provided:
+ *
+ * Templates::Catalog::Chapter::Main
+ * Templates::Catalog::Chapter::Details
```

Comment:
I would keep lines 46-50, because the hooks are not the same like in `monograph_full.tpl`. I had changed hooks names, that it is possible to add content only to book or only to chapter page.

Reply:
:+1: